src/test/java/uk/gov/hmcts/reform/civil/service/search/TrialReadyCheckSearchServiceTest.java

```diff
@@ -27,6 +27,7 @@ protected Query buildQuery(int fromValue) {
                 .must(rangeQuery("data.hearingDate").lt(LocalDate.now().atTime(LocalTime.MIN).plusWeeks(3)
                                                             .toString()))
                 .must(boolQuery().must(matchQuery("state", "PREPARE_FOR_HEARING_CONDUCT_HEARING")))
+                .mustNot(matchQuery("data.allocatedTrack", "SMALL_CLAIM"))
```

Comment:
Here it's fine to have a string as you want to check that the value is "correct" so you don't want to tie yourself to a constant. Just writing so you don't end up modifying this one too.